console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2128,23 +2128,48 @@ createIotDevice("iotProjectFrance", {
 
 createIotDevice("iotProjectFrance", {
   deviceId: "12",
-  jsonData: JSON.stringify({ via: 11, ext: { summit: "Mt Blanc" } }),
+  jsonData: JSON.stringify({
+    via: ["device-1", "device-2"],
+    ext: { summit: "Mt Blanc" }
+  }),
   credentials: []
 });
 
 createIotDevice("iotProjectIndia", {
   deviceId: "20",
   enabled: true,
   viaGateway: false,
-  jsonData: JSON.stringify({ ext: { ocean: "indian" } }),
+  jsonData: JSON.stringify({
+    via: ["20", "21"],
```

Comment:
you can't have "viaGateway" set to false and have devices in the via field at the same time

Reply:
yes, I made changes you suggested.